controllers/setup.go

```diff
@@ -50,13 +50,16 @@ func (r *KymaReconciler) SetupWithManager(mgr ctrl.Manager,
 	options controller.Options, settings SetupUpSetting,
 ) error {
 	generationChangedPredicate := predicate.GenerationChangedPredicate{}
+	labelChangedPredicate := predicate.LabelChangedPredicate{}
+	annotationChangedPredicate := predicate.AnnotationChangedPredicate{}
```

Comment:
@ruanxin IMO we should react to changes in labels and annotations since they could be altered manually after Kyma creation so then we don't have to wait for the success interval till the next reconciliation, WDYT?

Reply:
Regarding reacting on manual change, I agreed on labels, but annotation is not designed for updated by user manually. That metadata in annotation should be managed by the operator. For instance, Kyma CR have the annotation to save crd generations, manifest CR has the annotation of `sync-oci-ref` to track oci layer id. 

If there is an update that happens, it anyway detected and processed by LM itself during reconciliation time, we don't need to put the item enqueue again immediately.